vita3k/gui/src/gui.cpp

```diff
@@ -63,8 +63,7 @@ void draw_info_message(GuiState &gui, EmuEnvState &emuenv) {
         ImGui::SetNextWindowPos(ImVec2(emuenv.viewport_pos.x + (display_size.x / 2) - (WINDOW_SIZE.x / 2.f), emuenv.viewport_pos.y + (display_size.y / 2.f) - (WINDOW_SIZE.y / 2.f)), ImGuiCond_Always);
         ImGui::PushStyleVar(ImGuiStyleVar_ChildRounding, 10.f * SCALE.x);
         ImGui::BeginChild("##info", WINDOW_SIZE, true, ImGuiWindowFlags_NoSavedSettings | ImGuiWindowFlags_NoDecoration);
-        auto title = fmt::format("{}", spdlog::level::to_string_view(gui.info_message.level));
-        title[0] = std::toupper(title[0]);
+        auto title = gui.lang.app_context.main["information"];
```

Comment:
Revert this, is not string informations att all
this header of dialog indicate gui.info_message.level, relate level error get

Reply:
but the spdlog is display in log, should be not display in gui, if u do not like it, i revert it, Mr. Zangetsu

Reply:
Yes, I know, it's a short word like "info", it's just to quickly tell the user in the header if "info" = good, "error" = bad, don't forget not, this dialog, totally replace the log

Sorry for this